src/test/coll.rs

```diff
@@ -753,3 +756,27 @@ async fn no_read_preference_to_standalone() {
 
     assert!(!command_started.command.contains_key("$readPreference"));
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+#[function_name::named]
+async fn watch_test() {
+    let _guard = LOCK.run_concurrently().await;
+
+    let client = EventClient::new().await;
+    if client.server_version_lt(4, 0) || client.is_standalone() {
```

Comment:
change streams were introdued in 3.6, is there any reason we're version gating here?

Reply:
my bad, collections work on 3.6+. fixed